src/translate/translateConfig.js

```diff
@@ -0,0 +1,28 @@
+import angular from 'angular';
+
+export default angular.module('translate')
+    .config(['$translateProvider', ($translateProvider) => {
+        $translateProvider.translations('en', {
+            TITLE: 'tutorials point',
```

Comment:
Translation module should have a single source of truth for translations. It is a good idea to keep a JSON for ease of use, but we need to find a way to extract the data instead of manually copying it. I suggest either doing it with Gulp and/or fetching it from server. Please add this to todo list on Trello.

Reply:
Task added on the trello board.